src/avatar-selector.js

```diff
@@ -26,22 +25,16 @@ addLocaleData([...en]);
 
 registerTelemetry();
 
+const hash = new URLSearchParams(location.hash.replace(/^#/, "?"));
```

Comment:
it [won't break](https://caniuse.com/#search=URLSearchParams) for any existing users of Hubs, so feel free to ignore, but figured I'd mention anyway: `URLSearchParams` isn't supported in IE 11 (which I know is not a supported browser), and it was only recently [added in Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8993198/) on April 10, 2018.

Reply:
not sure if it's any more legible but you could do `(window.location.hash || "#").substr(1)`

Reply:
> it won't break for any existing users of Hubs, so feel free to ignore, but figured I'd mention anyway: URLSearchParams isn't supported in IE 11 (which I know is not a supported browser), and it was only recently added in Edge on April 10, 2018.

Yeah, in general I think we are only interested in supporting the following:

- Latest stable Edge, Chrome, and Firefox on Linux, Windows, and OS X
- Latest stable Chrome and Firefox on Android w/VR
- Reasonably recent Android and iOS browsers w/magic window

So I'm not concerned about support.

Personally, I like the regex better, because it's clear to me what it does without having to think to myself "if I call `substr` with one argument is it the start or the end and is it inclusive or exclusive", but I guess it's going to depend on whether you remember what regex `^` is.